scripts/travis.install.sh

```diff
@@ -6,25 +6,5 @@ shopt -s expand_aliases
 # TODO: We know that $DEV_LIB_PATH is _this_ directory, use that directly.
 source "$DEV_LIB_PATH/check-diff.sh"
 
-# The following should be temporary. See <https://core.trac.wordpress.org/ticket/40086>, <https://core.trac.wordpress.org/ticket/39822>.
```

Comment:
@westonruter Do you remember why this was added here instead of the PHPUnit related section of `install_tools`?

Reply:
This was introduced in https://github.com/xwp/wp-dev-lib/pull/221. I believe I added it here as a quick workaround, intending that it would be removed shortly thereafter. And then 2 years passed.